src/migrations/20220311135140-update-topics-table.js

```diff
@@ -0,0 +1,90 @@
+const TOPICS = [
+  'Behavioral / Mental Health / Trauma',
+  'Child Assessment, Development, Screening',
+  'CLASS: Classroom Organization',
+  'CLASS: Emotional Support',
+  'CLASS: Instructional Support',
+  'Coaching',
+  'Communication',
+  'Community and Self-Assessment',
+  'Culture & Language',
+  'Curriculum (Instructional or Parenting)',
+  'Data and Evaluation',
+  'ERSEA',
+  'Environmental Health and Safety / EPRR',
+  'Equity',
+  'Facilities',
+  'Family Support Services',
+  'Fiscal / Budget',
+  'Five-Year Grant',
+  'Home Visiting',
+  'Human Resources',
+  'Leadership / Governance',
+  'Learning Environments',
+  'Nutrition',
+  'Oral Health',
+  'Parent and Family Engagement',
+  'Partnerships and Community Engagement',
+  'Physical Health and Screenings',
+  'Pregnancy Services / Expectant Families',
+  'Program Planning and Services',
+  'Quality Improvement Plan / QIP',
+  'Recordkeeping and Reporting',
+  'Safety Practices',
+  'Staff Wellness',
+  'Teaching Practices / Teacher-Child Interactions',
+  'Technology and Information Systems',
+  'Transition Practices',
+  'Transportation',
+];
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => queryInterface.sequelize.transaction(
+    async (transaction) => {
+    // create an "isSelectable" column, denoting that this topic
```

Comment:
Might be better to use https://sequelize.org/v6/manual/paranoid.html

Reply:
You're suggesting not bothering with the isSelectable column, just making the model paranoid and deleting everything that would be unselectable?

Reply:
I think we can hard delete any topics not currently in use (because no other models reference this table). In the future when we won't be able to hard delete sequelize has a built in way of handling soft deletes that we use elsewhere, would be nice to handle it in the same way unless there is something I'm not thinking about. If we use paranoid we don't have to worry about adding `where: { isSelectable: true }` to our queries.